api/src/wfl/module/arrays.clj

```diff
@@ -0,0 +1,224 @@
+(ns wfl.module.arrays
+  "Process Arrays for the Broad Genomics Platform."
+  (:require [clojure.java.io :as io]
+            [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [wfl.environments :as env]
+            [wfl.api.workloads :as workloads :refer [defoverload]]
+            [wfl.jdbc :as jdbc]
+            [wfl.module.all :as all]
+            [wfl.module.batch :as batch]
+            [wfl.references :as references]
+            [wfl.service.cromwell :as cromwell]
+            [wfl.service.gcs :as gcs]
+            [wfl.service.postgres :as postgres]
+            [wfl.util :as util]
+            [wfl.wdl :as wdl]
+            [wfl.wfl :as wfl]
+            [clojure.data.json :as json])
+  (:import [java.time OffsetDateTime]))
+
+(def pipeline "GPArrays")
+
+(def workflow-wdl
+  "The top-level WDL file and its version."
+  {:release "Arrays_v2.3.0"
+   :top     "pipelines/broad/arrays/single_sample/Arrays.wdl"})
+
+(def cromwell-label-map
+  "The WDL label applied to Cromwell metadata."
+  {(keyword wfl/the-name)
+   pipeline})
+
+(def cromwell-label
+  "The WDL label applied to Cromwell metadata."
+  (let [[key value] (first cromwell-label-map)]
+    (str (name key) ":" value)))
+
+(def fingerprinting
+  "Fingerprinting inputs for arrays."
+  {:haplotype_database_file              "gs://gcp-public-data--broad-references/hg19/v0/Homo_sapiens_assembly19.haplotype_database.txt"
+   :variant_rsids_file                   "gs://broad-references-private/hg19/v0/Homo_sapiens_assembly19.haplotype_database.snps.list"})
+
+(def other-inputs
+  "Miscellaneous inputs for arrays."
+  {:contamination_controls_vcf       nil
+   :subsampled_metrics_interval_list nil
+   :disk_size                        100
+   :preemptible_tries                3})
```

Comment:
Why `nil`?

Reply:
Honestly I'm not sure...I just copied this from the AOU module, but I can ask the Lantern team if it should be different

Reply:
According to George they're not provided at the moment: https://broadinstitute.slack.com/archives/CESEYJW9W/p1605646889170800?thread_ts=1605645572.170700&cid=CESEYJW9W